client/asset/btc/spv_wrapper.go

```diff
@@ -1158,7 +1158,25 @@ func (w *spvWallet) moveWalletData(backupDir string) error {
 		return err
 	}
 	backupFolder := filepath.Join(backupDir, timeString)
-	return os.Rename(w.dir, backupFolder)
+	if err := os.Rename(w.dir, backupFolder); err != nil {
+		return err
+	}
+	// Copy wallet logs.
+	if err := w.copyWalletLogFile(filepath.Join(backupFolder, logDirName)); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (w *spvWallet) copyWalletLogFile(dstDir string) error {
+	if err := os.MkdirAll(dstDir, 0744); err != nil {
+		return err
+	}
+	logBytes, err := os.ReadFile(w.logFilePath())
+	if err != nil {
+		return err
+	}
+	return os.WriteFile(filepath.Join(dstDir, logFileName), logBytes, 0744)
```

Comment:
Is this sufficient? We have a log rotator that gzips logs periodically and performs some amount of truncation (starts from scratch?). For example, here is what my current mainnet **assetdb/btc/mainnet/logs** looks like.
![image](https://user-images.githubusercontent.com/6109680/204164838-86916bed-633c-4efa-bf16-831332a6ebb4.png)

Reply:
Aren't we also going to have a collision between network at **assetdb/btc/spvlogs**?  That's the new log path right?

Reply:
> Is this sufficient?

Ah, maybe not. We would need to copy the whole directory content.